engine/cmd/master/main.go

```diff
@@ -58,6 +59,7 @@ func main() {
 		os.Exit(2)
 	}
 	version.LogVersionInfo()
+	gin.SetMode(gin.ReleaseMode)
```

Comment:
Do we need `DebugMode` and `TestMode` for future debug or test?

Reply:
I think we can use environment variable to set debug or test mode and set release mode by default. I will fix later.